### PR TITLE
Tretikoff/kotlin styled next benchmarks

### PR DESCRIPTION
**Pull request by tretikoff**

### CODE REVIEW

File: kotlin-styled-next/src/jsTest/kotlin/test/RulesBuild.kt [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+import react.RProps
+import react.fc
+import runTest
+import styled.StyleSheet
+import styled.css
+import styled.styledSpan
+import waitForAnimationFrame
+import kotlin.test.*
+
+private object SimpleStyleSheet : StyleSheet("testStyleSheet") {
```
Comment (Leonya) [inline kotlin-styled-next/src/jsTest/kotlin/test/RulesBuild.kt:17]: testStyleSheet -> SimpleStyleSheet

File: kotlin-styled-next/src/jsTest/kotlin/test/RulesBuild.kt [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+    val specificProperty by css {
+        specific {
+            padding(80.px)
+            border = "none"
+        }
+    }
+}
+
+
+private object StaticStyleSheet : StyleSheet("testStyleSheet", isStatic = true) {
```
Comment (Leonya) [inline kotlin-styled-next/src/jsTest/kotlin/test/RulesBuild.kt:32]: testStyleSheet -> StaticStyleSheet

File: kotlin-styled-next/build.gradle.kts [hunk 3]
```diff
@@ -1,15 +1,104 @@
...
+    kotlin("multiplatform")
     `publish-conventions`
 }
 
-dependencies {
-    api(project(":kotlin-extensions"))
-    api(project(":kotlin-css"))
-    api(project(":kotlin-react"))
-    api(project(":kotlin-react-dom"))
+repositories {
```
Comment (Leonya) [inline kotlin-styled-next/build.gradle.kts:6]: Not sure what all these repositories (except mavenCentral) are needed for, seems like this section can be dropped.

File: kotlin-styled-next/build.gradle.kts [hunk 4]
```diff
@@ -1,15 +1,104 @@
...
+            }
+            tasks.withType<org.jetbrains.kotlin.gradle.dsl.KotlinCompile<*>>().configureEach {
+                kotlinOptions.freeCompilerArgs += "-Xopt-in=kotlin.RequiresOptIn"
+            }
+            val printBundleSize by tasks.registering {
+                dependsOn(tasks.named("jsBrowserDistribution"))
+                doLast {
+                    val jsFile = buildDir.resolve("distributions/web-benchmark-core.js")
+                    val size = jsFile.length()
+                    println("##teamcity[buildStatisticValue key='landingPageBundleSize' value='$size']")
```
Comment (Leonya) [inline kotlin-styled-next/build.gradle.kts:59]: What landing page?

**Review by Leonya — CHANGES_REQUESTED** [review]

**Comment by tretikoff** [comment]
Закрываю в пользу https://github.com/JetBrains/kotlin-wrappers/pull/576

**Pull request closed without merging**